src/command.ts

```diff
@@ -1,97 +1,95 @@
-import {
+import type {
   BotCommand,
   BotCommandScope,
-  Chat,
-  ChatTypeContext,
-  CommandMiddleware,
-  Composer,
+  BotCommandScopeAllChatAdministrators,
+  BotCommandScopeAllGroupChats,
+  BotCommandScopeAllPrivateChats,
+  ChatTypeMiddleware,
   Context,
-  match,
   Middleware,
-  P,
+  MiddlewareObj,
 } from "./deps.deno.ts";
+import { Composer, match, P } from "./deps.deno.ts";
```

Comment:
nit: merge the two imports into one

Reply:
I hate that the non-type imports end up in the middle :C
![image](https://github.com/grammyjs/commands/assets/3948961/985dea8b-4914-4e69-bfe5-1e9a15b2ee27)

Reply:
But done anyway.
I just wanna get this merged so I can use it on a bot hehe